src/components/AlternativeVerticals.tsx

```diff
@@ -17,27 +17,17 @@ export interface AlternativeVerticalsCssClasses {
   alternativeVerticals___loading?: string,
   noResultsText?: string,
   categoriesText?: string,
-  suggestions?: string,
-  suggestionList?: string,
   suggestion?: string,
-  suggestionButton?: string,
-  verticalIcon?: string,
-  verticalLink?: string,
-  allCategoriesLink?: string
+  verticalIcon?: string
 }
 
 const builtInCssClasses: Readonly<AlternativeVerticalsCssClasses> = {
   container: 'flex flex-col justify-between border rounded-lg mb-4 p-4 shadow-sm',
   alternativeVerticals___loading: 'opacity-50',
```

Comment:
nit: can we get product's thoughts on potentially updating these two names to be consistent? like `container` and `container_loading` OR `alternativeVerticals` and `alternativeVerticals_loading`?